src/components/ErrorBanner.tsx

```diff
@@ -0,0 +1,74 @@
+import * as React from 'react'
+import { Text, View, StyleSheet } from 'react-native'
+import Modal from 'src/components/Modal'
+import colors from 'src/styles/colors'
+import { Spacing } from 'src/styles/styles'
+import ExclamationMark from 'src/icons/ExclamationMark'
+import { useTranslation } from 'react-i18next'
+import { typeScale } from 'src/styles/fonts'
+
+interface Props {
+  text: string
+  isVisible: boolean
+  testID: string
+  onDismiss: () => void
+}
+
+export default function ErrorBanner({ text, isVisible, testID, onDismiss }: Props) {
+  const { t } = useTranslation()
+
+  return (
+    <Modal
+      modalStyle={styles.modalStyle}
+      style={styles.modalContentStyle}
+      isVisible={isVisible}
+      testID={testID}
+      onModalHide={onDismiss}
+      onBackgroundPress={onDismiss}
+    >
+      <View>
+        <View style={styles.titleRow}>
```

Comment:
I believe this is the same style as https://github.com/valora-inc/wallet/blob/main/src/components/InLineNotification.tsx . Can you confirm and try to use that component if possible?

Reply:
omg, talk about reinventing the wheel... i could have sworn we had this already implemented, i just couldn't find it.. thanks for the callout!!